### PR TITLE
Add style for having one or two images in a row

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +159,22 @@ It's good practice to surround URLs: `<http://some.url>`. GitHub will linkify ev
 
 If you are modifying a template (`*.mdpp` file), don't forget to run `./precheckin.sh` before committing.
 
+#### Images
+
+Getting images to look nice can be tricky. There is a helper style which allows adding two images on a single row, while gracefully falling back to displaying images on two rows if there isn't sufficient space. You can use it as follows:
+```
+<div class="flex-images" markdown="1">
+* <a href="example1.png"><img src="example1.png" alt="Example image 1"></a><span class="md_figcaption">Example image 1</span>
+* <a href="example2.png"><img src="example2.png" alt="Example image 2"></a><span class="md_figcaption">Example image 2</span>
```
Comment (sledges) [inline README.md:168]: Can be split, so it's easier to find/edit the caption:
```suggestion
* <a href="example1.png"><img src="example1.png" alt="Example image 1"></a>
  <span class="md_figcaption">
    Example image 1
  </span>
* <a href="example2.png"><img src="example2.png" alt="Example image 2"></a>
  <span class="md_figcaption">
    Example image 2
  </span>
```

File: README.md [hunk 2]
```diff
@@ -158,6 +159,22 @@ It's good practice to surround URLs: `<http://some.url>`. GitHub will linkify ev
...
+</div>
+```
+You can use the style with just one image as well. If you have only one image, which happens to be a narrow one (e.g. screenshot from a device), you can use additional `narrow-image` style on the anchor tag:
+```
+<div class="flex-images" markdown="1">
+* <a href="example3.png" class="narrow-image"><img src="example3.png" alt="Example narrow image"></a><span class="md_figcaption">Example narrow image</span>
```
Comment (sledges) [inline README.md:174]: Same as above.
```suggestion
* <a href="example3.png" class="narrow-image"><img src="example3.png" alt="Example narrow image"></a>
  <span class="md_figcaption">
    Example narrow image
  </span>
```

**Review by sledges — COMMENTED** [review]
Looks great! Couple of non-major comments.

**Review by sledges — APPROVED** [review]

**Review by jovirkku — COMMENTED** [review]
I had a question on th asterisk characters in the beginning of the "a href" lines. Why use bullets there? But there was a reply that they do not make bullets appear on the final doc pages.

**Comment by vigejolla** [comment]
> I had a question on th asterisk characters in the beginning of the "a href" lines. Why use bullets there? But there was a reply that they do not make bullets appear on the final doc pages.

The reason is that we (ab)use unordered lists here. It was just an easy way of having the images and captions inside one html element, without having to actually write html. Unfortunately there's still some html left, but at least it's less than what it used to be.

**Pull request merged**